Remotion/Web/Core/UI/SmartPageImplementation/SmartPageInfo.cs

```diff
@@ -295,7 +295,7 @@ private void Page_PreRenderComplete (object? sender, EventArgs eventArgs)
 
     private void PreRenderSmartPage ()
     {
-      var smartPageTokenValue = (long)(DateTime.UtcNow - s_pageTokenStartDate).TotalMilliseconds;
+      var smartPageTokenValue = (DateTime.UtcNow - s_pageTokenStartDate).Ticks;
```

Comment:
before I approve, please check if increasing the token-length has any adverse effects on the browser state management. We are using the token as part of a key for the clientside storage API and a longer key will increase memory consumption. we need to make sure that the increase is negligible. Please check the avaialble discussio in the original tickets on this and do a quick performance test.